build.gradle

```diff
@@ -67,4 +67,14 @@ subprojects {
       showCauses true
     }
   }
+
+  configurations.all {
+    resolutionStrategy {
+      eachDependency { DependencyResolveDetails details ->
+        if("org.jetbrains.kotlin" == details.requested.group) {
+          details.useVersion versions.kotlin
+        }
+      }
+    }
+  }
```

Comment:
this shouldn't be necessary.  what does this fix?

Reply:
IIRC some dependencies are bringing in different versions of Kotlin which was causing some issue. This causes only the specified version of Kotlin to be used.

Reply:
Gradle should sort this out dependency versions.  Mind pasting the specific error?

Reply:
It's possible that it was just this warning:

```
w: Runtime JAR files in the classpath have the version 1.4, which is older than the API version 1.5. Consider using the runtime of version 1.5, or pass '-api-version 1.4' explicitly to restrict the available APIs to the runtime of version 1.4. You can also pass '-language-version 1.4' instead, which will restrict not only the APIs to the specified version, but also the language features
w: /home/eli/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.32/ce852b166d97f0f1991b5130c2bb02e2ef6c554e/kotlin-reflect-1.4.32.jar: Runtime JAR file has version 1.4 which is older than required for API version 1.5
w: /home/eli/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.32/ef50bfa2c0491a11dcc35d9822edbfd6170e1ea2/kotlin-stdlib-common-1.4.32.jar: Runtime JAR file has version 1.4 which is older than required for API version 1.5
w: /home/eli/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.32/3546900a3ebff0c43f31190baf87a9220e37b7ea/kotlin-stdlib-jdk7-1.4.32.jar: Runtime JAR file has version 1.4 which is older than required for API version 1.5
w: /home/eli/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.4.32/3302f9ec8a5c1ed220781dbd37770072549bd333/kotlin-stdlib-jdk8-1.4.32.jar: Runtime JAR file has version 1.4 which is older than required for API version 1.5
w: /home/eli/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.32/461367948840adbb0839c51d91ed74ef4a9ccb52/kotlin-stdlib-1.4.32.jar: Runtime JAR file has version 1.4 which is older than required for API version 1.5

```

Reply:
Ah ok.  If so, this can safely be ignored and the snippet above removed.

Reply:
Removed and updated PR